_posts/2017-10-10-0.2-announcement.md

```diff
@@ -6,6 +6,7 @@ permalink: blog/istio-0.2-announcement.html
 attribution: The Istio Team
 layout: post
 type: markdown
+toc: false
```

Comment:
true ?

Reply:
Nope, that page I want off. I turned the TOC on for all posts by default, but this post doesn't need one so it is explicitly excluded.